app/_hub/kong-inc/json-threat-protection/overview/_index.md

```diff
@@ -0,0 +1,234 @@
+---
+nav_title: Overview
+---
+
+The JSON Threat Protection plugin provides security validation on various aspects of JSON to ensure that the incoming JSON in requests adheres to the security policy limits.
+
+It will check the following points:
```

Comment:
Suggest following reword.

The JSON Threat Protection plugin provides security validation against various aspects of JSON structure. The plugin validates the incoming JSON request body to ensure the payload adheres to the policy limits. Requests violating the policy will be considered malicious. The plugin can be configured to drop such requests thus protecting it from reaching the service. Optionally, the plugin can operate in tap mode to monitor the traffic

The plugin will check the following limits:

Reply:
I’ve updated the doc and added some additional content.